ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.xx.x branch released xxxx-xx-xx
+
+Bugfixes
+   * Fix C90 compatibility issue in hkdf.c. Contributed by Brian J Murray.
+   * Fix issue if salt = NULL and salt_len !=0 in mbedtls_hkdf_extract(). Contributed by Brian J Murray.
```

Comment:
Please comply to the line length limit of 80 characters. Also, if I may suggest a slight rewording:

> * Change the default behaviour of mbedtls_hkdf_extract() to return an error  
>   when calling with a NULL salt and non-zero salt_len. Contributed by  
>   Brian J Murray.